app/stac_api/models.py

```diff
@@ -137,8 +139,18 @@ class Collection(models.Model):
     # furthermore GeoDjango and its functionality will be used for that.
     # TODO: overwrite items save() function accordingly
     # suggestions of fields to be auto-populated:
-    extent = models.JSONField(
-        default=get_default_extent_value, encoder=DjangoJSONEncoder, editable=False
+    # extent = models.JSONField(
+    #    default=get_default_extent_value, encoder=DjangoJSONEncoder, editable=False
+    # )
+    start_date = models.DateTimeField(editable=False, null=True, blank=True)
+    end_date = models.DateTimeField(editable=False, null=True, blank=True)
```

Comment:
maybe name them `start_datetime` and `end_datetime` (same naming as in the item). 

I'm just wondering if it would make sense to mark all these 'cache' fields with e.g. a `cache_*` prefix... just to clearly indicate which values are actually properties of the Model itself and which values are just precalculated (i.e. cached) values from other models.

Reply:
makes sense, both. So it will be `cache_start_datetime` and `cache_end_datetime`

Reply:
done